client/src/helpers/strings/wise.ts

```diff
@@ -17,40 +17,37 @@ const strings: PlatformStrings = {
 
   // Notarizations Instructions
   NO_NOTARIZATIONS_ERROR: `
-    No Wise tag requests found.
-    Please follow instructions in the extension to notarize an existing Wise account tag in the account page.
+    No Wise tag proofs found.
+    Please follow instructions in the extension to prove an existing Wise account.
   `,
   NO_TRANSFER_NOTARIZATIONS_ERROR: `
-    No Wise transfer requests found.
-    Please follow instructions in the extension to notarize an existing Wise payment in the transactions page.
+    No Wise transfer proofs found.
+    Please follow instructions in the extension to prove an existing Wise payment.
   `,
 
   // New Registration
   REGISTRATION_INSTRUCTIONS: `
-    You must register in order to use ZKP2P. Registration requires a notarized message from Wise, which is used
-    to prove you own a Wise account. Your Wise ID is hashed to conceal your identity.
+    You must register in order to use ZKP2P. Click on Register above to initiate registration.
```

Comment:
The second sentence is unnecessary. If there isn't anymore meaningful context we want to share, we can consider removing the component that displays this string altogether and stick it into a tooltip.

Reply:
If we remove entirely, this doesn't look very good.
<img width="701" alt="Screenshot 2024-04-08 at 12 15 50 AM" src="https://github.com/zkp2p/zk-p2p/assets/73331595/c220fe71-df6a-4f0f-bb64-11437aabb94e">

Reply:
Agreed, I mistook this for the instruction section inside the proving flow. Consider some variation of: `You must register in order to you ZKP2P. Provide proof of a valid Wise account.`

Reply:
Added in 4a5384f